doc/gettingstarted.md

```diff
@@ -87,11 +90,11 @@ A link will be provided to the OAuth login page where you can login using the cr
 By default, the stack allows unregistered gateways to connect, but without providing a default band. As such, it is highly recommended that each gateway is registered:
 
 ```bash
-$ docker-compose exec stack ttn-lw-cli gateway create gtw1 --user-id admin --frequency_plan_id EU_863_870 --gateway-eui 00800000A00009EF
+$ ttn-lw-cli gateway create gtw1 --user-id admin --frequency_plan_id EU_863_870 --gateway-eui 00800000A00009EF
```

Comment:
```suggestion
$ ttn-lw-cli gateway create gtw1 --user-id admin --frequency_plan_id EU_863_870 --gateway-eui 00800000A00009EF --enforce-duty-cycle
```

We should probably introduce a defaults flags here too.

Reply:
I agree, but that deserves its own issue

Reply:
For sure